CMakeLists.txt

```diff
@@ -395,7 +441,29 @@ else()
 	)
 endif()
 
-install(FILES ${AdditionalInstallFiles} DESTINATION ${auxresourcesdir})
+if(WIN32)
+    SET(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}")
+    include(InstallRequiredSystemLibraries)
+
+    install(FILES ${AdditionalInstallFiles} DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}")
+
+    if(MSYS)
+        SET(QT_PLATFORM_PLUGIN_PREFIX ${Qt5Core_DIR}/../../../share/qt5/plugins/platforms CACHE STRING "Location of qt platform plugin for windows. Override when not using MinGW-cmake!")
+    else()
+        SET(QT_PLATFORM_PLUGIN_PREFIX ${Qt5Core_DIR}/../../../plugins/platforms CACHE STRING "Location of qt platform plugin for windows. Override if you have multiple paths in your cmake prefix path")
+    endif()
+
+    install(FILES "${QT_PLATFORM_PLUGIN_PREFIX}/qwindows.dll"
```

Comment:
is it possible to run the build without installing? it doesn't look like, so you might want to copy qt's platform plugins via a post-build step